src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -156,31 +71,72 @@ const SearchResults = (props) => {
 
     if ((searchState.searchQueryStatus.account !== 200 ||
         searchState.searchQueryStatus.course !== 200) && searchState.searchQueryStatus.status) {
-        return <Loading />
+        return <SearchResultsLoader
+            SearchResults={numberOfResults()}
+            accountPage={currentPage.account}
+            coursePage={currentPage.course}
+            SearchQuery={SearchQuery}/>
     }
 
     const MAX_PAGE = Math.ceil(numberOfResults() / 8);
-    const pageNumber = (apiPage, startPage) => {
-        const firstResultsPage = startPage === 0 && apiPage == 1;
-        const secondResultsPage = startPage !== 0 && apiPage == 1;
-        if (firstResultsPage) {
-            return 1;
-        } else if(secondResultsPage){
-            return 2;
-        } else if (startPage === 0){
-            return Number(apiPage)*2;
+
+    const setPageLimit = (resultsPageNumber) => {
+        return resultsPageNumber >= MAX_PAGE;
+    };
```

Comment:
1. This is not `set`ting anything, so the function should be renamed appropriately.
2. You can use implicit returns from arrow functions here:

    const isWithinPageLimit = (resultsPageNumber) => (resultsPageNumber >= MAX_PAGE);

(parentheses around the condition are optional but added for clarity).